plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/refactoring/preview/StringStreamEditor.java

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
It looks like `org.eclipse.che.api.languageserver.shared.util.CharStreamEditor` but it isn't a copy. We cant use `CharStreamEditor` in the client because GWT doesn't support `java.io.*`.